### DISCUSSION TITLE
What's the plans on 6.5?

### DISCUSSION THREAD

**Discussion by Wohntrends-Shop-Strunz** [shyim/shopware#95] [General]
We purposely didn't immediately switch over and waited for 6.5.1 to clean up the major issues, the changelog in the length of a novel says we did nothing wrong there. There are a few last plugins which have not migrated, either. However we'd like to make first tests on the development server, so the question is: Will there be a 6.5 image anytime soon, or should we rather switch to dockware in that case?

**Comment by shyim** [comment]
Have a look here :) https://github.com/FriendsOfShopware/production-docker
